src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -45,6 +45,10 @@ As part of the updates to Amplify JavaScript v6, we also re-evaluated our strate
 
 ### @aws-amplify/react-native
 
+import rnVersionCallout from '/src/fragments/common/react-native-version-deployment-target.mdx';
+
+<Fragments fragments={{ all: rnVersionCallout }} />
```

Comment:
This should probably be below the explanation of the `@aws-amplify/react-native` package since it directly references `@aws-amplify/react-native`?

Reply:
I was a little torn on the placement of this one when I was adding this, and I couldn't decide if it made more sense for this callout to exist at the beginning or end of the section. Nonetheless, moving this works for me :)